Programming_Problems 5

```diff
@@ -1,5 +1,7 @@
-let n = 17;
-let x = 3 + 6 + 9 + 12 + 15;
-console.log(n);
-console.log(x);
-console.log(n + x);
+var num = prompt("Number n = ?");
+var sum = 0;
+for (var i = 3; i <= num; i++) {
+   if (i % 3 === 0; || i % 5 ===0) {
```

Comment:
Consider making the order of operations clearer.

if((i % 3) === 0 || (i % 5) === 0) {